tests/PHPUnit/System/EcommerceOrderWithItemsTest.php

```diff
@@ -67,11 +67,24 @@ public function getApiForTesting()
                         'date' => $dateTime,
                         'periods' => array('day', 'week'),
                         'otherRequestParameters' => array('_leavePiwikCoreVariables' => 1),
-                        'segment' => 'pageUrl=@Another%20Product%20page',
+                        'segment' => 'pageUrl=@example.org%2Findex.htm',
```

Comment:
here I fixed the system test, technically should have been in a separate PR as it's un-related